engine/dlib/src/dmsdk/dlib/profile.h

```diff
@@ -505,8 +505,6 @@
     #define DM_PROFILE(name)
     #define DM_PROFILE_DYN(name, name_hash)
 
-    #define DM_COUNTER_DYN(counter_index, amount)
```

Comment:
There was no good way of doing this in Remotery, so I waited for the support of properties.